src/back/glsl/mod.rs

```diff
@@ -816,13 +820,27 @@ impl<'a, W: Write> Writer<'a, W> {
         if self.options.version.supports_explicit_locations() {
             if let Some(ref br) = global.binding {
                 match self.options.binding_map.get(br) {
-                    Some(binding) => write!(self.out, "layout(binding = {}) ", binding)?,
+                    Some(binding) => {
+                        let layout = if self.options.version.supports_std430_layout() {
+                            if let crate::StorageClass::Storage { .. } = global.class {
+                                "std430, "
```

Comment:
looks like we are going to be writing `layout(std430)` both here and at line 842 down from here?